src/compat/qsort.c

```diff
@@ -260,8 +260,8 @@ _quicksort (void *const pbase, size_t total_elems, size_t size,
 #endif /* !HAVE_QSORT_R_GLIBC */
 #else
 #if _MSC_VER
```

Comment:
But now no need to use a ``#if _MSC_VER `` since both branches do the same thing.

Reply:
True.  I cannot test this so best if you mess around until it is OK. Just made a change so hopefully working.